packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -1219,13 +1239,38 @@ export class FluidDataStoreRuntime
 		this.submit(DataStoreMessageType.ChannelOp, envelope, localOpMetadata);
 	}
 
+	/**
+	 * @remarks Implementation detail for the {@link FluidDataStoreRuntime.pendingOpCount} property
+	 */
+	// eslint-disable-next-line unicorn/consistent-function-scoping -- It's an IIFE, only invoked once
+	private readonly _pendingOpCountState = (() => {
```

Comment:
i think you should atleast pull the function into a free function. that will remove the eslint-disable, and make it a bit clearer.

Reply:
you could also consider having the object returned have a real getter and setter, so it would be like _pendingOpCountState.value, and then you don't need the additional getter setters

Reply:
I'm using the IIFE to hide `value` from possibly being modified apart from the getter/setter.

Reply:
Ohh I see, let me try that